### PR TITLE
Frontend/Backend - Rendering get request discoveries from MongoDB Objects

### PR DESCRIPTION
**Pull request by benjamin-ranft**

### CODE REVIEW

**Review by NilsMester — APPROVED** [review]
Mega! Das wird wat ganz großet!

File: backend/src/main/java/de/benjaminranft/spotown/security/SecurityConfig.java [hunk 1]
```diff
@@ -24,11 +24,6 @@ public SecurityConfig(MongoDbUserDetailsService userDetailsService, JwtAuthFilte
         this.jwtAuthFilter = jwtAuthFilter;
     }
 
-    @Override
```
Comment (christophersiem) [inline backend/src/main/java/de/benjaminranft/spotown/security/SecurityConfig.java:27]: why did you remove this method?
